source/cleaning_service.rb

```diff
@@ -0,0 +1,82 @@
+# astia au mai multe statii de curatare
+# un orar in care functioneaza => hash cu zilele saptamanii
+# masinile programate
+require_relative 'time_table'
+
+class CleaningService
+  # attr_accessor :station_count 
+  # attr_accessor :car_count
+  attr_accessor :time_table
+  attr_accessor :stations
+  attr_accessor :queue
```

Comment:
you can have all the accessors on the same line something like:

`attr_accessor :time_table, :stations, :queue`